### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by dmyroponomarov**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -7,4 +7,34 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0', () => {
+    expect(getHumanAge(0, 0))
+      .toStrictEqual([0, 0]);
+  });
+
+  test('boundary values check, minimum age returned', () => {
```
Comment (djtyrf312) [inline src/getHumanAge.test.js:16]:
```suggestion
  test('boundary values check, minimum age returned', () => {
```
you have an incorrect test name. We write test names like 'should return something when something' or 'should do something when something'. Check all your test names

**Review by djtyrf312 — CHANGES_REQUESTED** [review]

File: src/getHumanAge.test.js (hunk 1 above)
Comment (dmyroponomarov) [reply src/getHumanAge.test.js]: @djtyrf312 thanks so much 4 your time, I'm sorry that I slow down sometimes))

**Review by Oleksii-Siviolov — APPROVED** [review]
toEqual is enough in this task, but it is not a mistake. Good job.
